plugin-server/src/worker/ingestion/property-definitions-auto-discovery.ts

```diff
@@ -81,6 +81,14 @@ export const detectPropertyDefinitionTypes = (value: unknown, key: string): Prop
         return PropertyType.String
     }
 
+    if (/^\$feature\//i.test(key)) {
```

Comment:
if we want to get funky `indexOf` turns out to be way faster than checking a regex

https://www.measurethat.net/Benchmarks/ShowResult/433361

<img width="707" alt="Screenshot 2023-06-22 at 15 45 23" src="https://github.com/PostHog/posthog/assets/984817/f08b09a6-c80b-412c-8e67-be9ec05ad4ea">

Reply:
I don't like to use regexes where I can avoid them, so I love this suggestion, added!

Reply:
```suggestion
    if (key.toLowerCase().indexOf('$feature/') === 0) {
```

true even with `toLowerCase()` call https://www.measurethat.net/Benchmarks/ShowResult/433368

<img width="741" alt="Screenshot 2023-06-22 at 15 51 34" src="https://github.com/PostHog/posthog/assets/984817/95d539b7-850b-42ad-a836-8c42b648d636">